streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/api/Twitter.java

```diff
@@ -0,0 +1,521 @@
+        LOGGER.warn("RestCallException", e);
+      }
+    } catch (IOException e) {
+      LOGGER.warn("IOException", e);
+    } catch (URISyntaxException e) {
+      LOGGER.warn("URISyntaxException", e);
+    }
+    return new ArrayList<>();
+  }
+
+  @Override
+  public Tweet show(StatusesShowRequest parameters) {
+//  TODO: juneau-6.2.0-incubating
+//      Statuses restStatuses = restClient.getRemoteableProxy("/statuses/show.json", Statuses.class);
+//      Tweet result = restStatuses.show(parameters);
+//      return result;
+    try {
+      URIBuilder uriBuilder = new URIBuilder()
+          .setPath("/statuses/show.json");
+      if (Objects.nonNull(parameters.getId()) && StringUtils.isNotBlank(parameters.getId().toString())) {
+        uriBuilder.addParameter("id", parameters.getId().toString());
+      }
+      if (Objects.nonNull(parameters.getTrimUser()) && StringUtils.isNotBlank(parameters.getTrimUser().toString())) {
+        uriBuilder.addParameter("trim_user", parameters.getTrimUser().toString());
+      }
+      if (Objects.nonNull(parameters.getIncludeEntities()) && StringUtils.isNotBlank(parameters.getIncludeEntities().toString())) {
+        uriBuilder.addParameter("include_entities", parameters.getIncludeEntities().toString());
+      }
+      if (Objects.nonNull(parameters.getIncludeMyRetweet()) && StringUtils.isNotBlank(parameters.getIncludeMyRetweet().toString())) {
+        uriBuilder.addParameter("include_my_retweet", parameters.getIncludeMyRetweet().toString());
+      }
+      RestCall restCall = restClient.doGet(uriBuilder.build().toString());
+      try {
+        String restResponseEntity = restCall
+            .setRetryable(configuration.getRetryMax().intValue(), configuration.getRetrySleepMs().intValue(), new TwitterRetryHandler())
+            .getResponseAsString();
+        //Tweet result = restCall.getResponse(Tweet.class);
+        Tweet result = mapper.readValue(restResponseEntity, Tweet.class);
+        return result;
+      } catch (RestCallException e) {
+        LOGGER.warn("RestCallException", e);
+      }
+    } catch (IOException e) {
+      LOGGER.warn("IOException", e);
+    } catch (URISyntaxException e) {
+      LOGGER.warn("URISyntaxException", e);
+    }
+    return null;
+  }
+
+  @Override
+  public FriendsIdsResponse ids(FriendsIdsRequest parameters) {
+//  TODO: juneau-6.2.0-incubating
+//      Friends restFriends = restClient.getRemoteableProxy("/friends/ids.json", Friends.class);
+//      FriendsIdsResponse result = restFriends.ids(parameters);
+//      return result;
+    try {
+      URIBuilder uriBuilder = new URIBuilder()
+          .setPath("/friends/ids.json");
+      if( Objects.nonNull(parameters.getCount()) && StringUtils.isNotBlank(parameters.getCount().toString())) {
+        uriBuilder.addParameter("count", parameters.getCount().toString());
+      }
+      if( Objects.nonNull(parameters.getCurser()) && StringUtils.isNotBlank(parameters.getCurser().toString())) {
```

Comment:
nice catch. that was wrong a bunch of places.  all fixed now and i watch the providers page as expected.